README.md

```diff
@@ -36,7 +36,7 @@ Windows is not tested and not supported for now.
 - Install PostgreSQL 9.5.x of later. (9.6 preferred) 
   On Debian distros use: `sudo apt-get install postgresql-9.5-dev`
```

Comment:
On Ubuntu 16.04: This statement gives me the error: 
`Unable to locate package postgresql-9.5-dev`
You can change it `postgresql-9.5`

Reply:
sure 👍